win32ss/gdi/eng/device.c

```diff
@@ -32,6 +32,133 @@ InitDeviceImpl(VOID)
     return STATUS_SUCCESS;
 }
 
+static
+BOOLEAN
+EngpHasVgaDriver(
+    IN PGRAPHICS_DEVICE pGraphicsDevice)
```

Comment:
I don't care much for that stuff myself, but maybe others: Would using SAL2 make sense here?
If you say "no", or "I am too lazy" or "would mismatch the other functions in here", I would be fine with that.

Reply:
Done